check/configuration/supported/bandit.md

```diff
@@ -0,0 +1,22 @@
+---
+description: Bandit- A Security Linter for Python 
+---
+
+# Bandit
+
+Bandit is a linter for Python.
+
+You can enable the Bandit plugin with
+
+```shell
+trunk check enable bandit
+```
+
+# Settings
+
+Bandit uses the same config files as the 
+upstream [Bandit]() project, so you can continue to use any
+existing configuration files (ex: `.bandit`).
```

Comment:
I need to sync up with Dubz about his experience using bandit since it looks like we may have some config issues